saleor/graphql/product/mutations/product/product_update.py

```diff
@@ -53,7 +52,6 @@ def clean_attributes(
     @classmethod
     def post_save_action(cls, info: ResolveInfo, instance, cleaned_input):
         product = models.Product.objects.prefetched_for_webhook().get(pk=instance.pk)
-        update_product_search_vector(instance)
```

Comment:
Can you add a test case for this mutation? It's quite hidden that `is_dirty` is set to `True`

Reply:
I've added a check for this field to `product_update` and `product_create tests`.